PollApp.ts

```diff
@@ -30,35 +30,97 @@ export class PollApp extends App implements IUIKitInteractionHandler {
     public async executeViewSubmitHandler(context: UIKitViewSubmitInteractionContext, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify) {
         const data = context.getInteractionData();
 
-        const { state }: {
-            state: {
-                poll: {
-                    question: string,
-                    [option: string]: string,
-                },
-                config?: {
-                    mode?: string,
-                    visibility?: string,
-                },
-            },
-        } = data.view as any;
-
-        if (!state) {
-            return context.getInteractionResponder().viewErrorResponse({
-                viewId: data.view.id,
-                errors: {
-                    question: 'Error creating poll',
-                },
-            });
-        }
+        const title = data.view.title.text;
+
+        switch(title) {
```

Comment:
I don't think title is the best property to understand the type of the modal. We have other properties like `view.id` which we usually use to for this purpose. We append a constant string to the id like `create_poll_form-${ uuid() }` while creating the view and them over here in `executeViewSubmitHandler` we determine the modal action via the prefix we'd earlier attached